Userland/Applications/PixelPaint/BrushTool.cpp

```diff
@@ -25,11 +25,15 @@ BrushTool::~BrushTool()
 {
 }
 
-void BrushTool::on_mousedown(Layer&, GUI::MouseEvent& event, GUI::MouseEvent&)
+void BrushTool::on_mousedown(Layer& layer, GUI::MouseEvent& event, GUI::MouseEvent&)
 {
     if (event.button() != GUI::MouseButton::Left && event.button() != GUI::MouseButton::Right)
         return;
 
+    for (int i = 0; i < 10; i++)
```

Comment:
What is the significance of `10` here? Where does it come from?
Also please use prefix increment (`++i`)

Reply:
Since one call to draw_point() draws quite a light stroke this was an approximation for about 4 clicks to full "opacity". I could change it to a named constant instead for clarity. It's also possible to derive it from m_hardness divided by 4 or 5 for a similar effect.